sdk/java-sdk/sdk/src/main/java/spec/sdk/runtime/v1/domain/file/GetFileRequest.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2021 Layotto Authors
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package spec.sdk.runtime.v1.domain.file;
+
+import java.io.OutputStream;
+import java.util.Map;
+
+public class GetFileRequest {
+
+    public String              storeName;
+    public String              filename;
```

Comment:
@wenxuwan I found that the field in the proto file is `name`,not `filename`. So which variable name should he use ?

Reply:
for sdk, fileName is ok

Reply:
@wenxuwan refact done.